src/html/htmlwin.cpp

```diff
@@ -1184,7 +1184,119 @@ void wxHtmlWindow::OnPaint(wxPaintEvent& WXUNUSED(event))
     }
 }
 
+namespace
+{
+
+// Returns true if leftCell is an ancestor of rightCell.
+bool IsAncestor(const wxHtmlCell* leftCell, const wxHtmlCell* rightCell)
+{
+    for ( const wxHtmlCell* parent = rightCell->GetParent();
+          parent; parent = parent->GetParent() )
```

Comment:
```suggestion
          parent;
          parent = parent->GetParent() )
```
This is just a minor style point, but I think we only ever have loops written on 1 line or on 3, but not 2.

(also below)